docs/tutorial.md

```diff
@@ -0,0 +1,182 @@
+# Oria Operator Tutorial
+
+An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.21.0 or newer
+- User authorized with `cluster-admin` permissions so that you can install an operator, CRDs and change rbac.
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Overview
+
+We will create a sample project to let you know how it works and this sample will:
+
+- Create a `memcached-operator` using below mentioned steps. For more infomation refer to this [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/)
+- Generate the manifests using `config/default`
+- Comment out `ClusterRole` and `ClusterRoleBinding`. 
+- Create a `ClusterRole` template in a `ScopeTemplate` and associate it with a `ScopeInstance`
+- Run the operator, the `memcached-operator` will start complaining about `RBAC`.
+- Run the `oria-operator`, it will create `ClusterRole` and `ClusterRoleBinding`. This will make sure that `memcached-operator` will not complain about `RBAC`. 
+
+### Install and run oria-operator
+
+Clone the [github repo](https://github.com/operator-framework/oria-operator.git) and  run the below commands on the root of the directory.
+
+```
+$ make install
+
+$ make run
+```
```

Comment:
To make it easy for people following along with the tutorial we should add the command needed to clone the repo. We can also combine the `make install` and `make run` commands into a single command (`make install run`). I would also put each command in it's own code block without the `$` so that they can copy the command easily and just paste it into the terminal.

I think it might also be worth adding a note at the end of this section that this will be a process that runs in the foreground and that they will need to open a new terminal window/tab to continue with the rest of the tutorial.

Reply:
The above process runs in the foreground. You will need to open a new terminal window/tab to continue with the rest of the tutorial.

Reply:
updated